packages/app/src/systems/Vault/services/VaultServer.ts

```diff
@@ -80,6 +80,19 @@ export class VaultServer extends EventEmitter {
       }
       this.server.addMethod(methodName, this[methodName].bind(this));
     });
+
+    this.manager.on('unlock', () => {
+      chrome.runtime.sendMessage({
```

Comment:
You can use this.server.notify to send messages from here to the client. https://www.npmjs.com/package/json-rpc-2.0

This is better to ensure the events are submitted in a standard way, avoid the use directly of the chrome.runtime

Reply:
@luizstacio 
Thank you for the suggestion to use this.server.notify for event submissions. However, it's important to note that JSON-RPC 2.0, while supporting structured message handling, does not inherently manage the transport layer. To enable this.server.notify to send messages from the server to the client, we would need a [bi-directional communication setup](https://www.npmjs.com/package/json-rpc-2.0#bi-directional-json-rpc), such as a custom WebSocket server and client. This setup would handle the transport of JSON-RPC messages bi-directionally, but would require significant development and maintenance efforts.

Given the complexity of implementing a custom WebSocket infrastructure, should we proceed with this approach considering the potential overhead?